### ISSUE TITLE
chore: add support for ARM architectures

### ISSUE THREAD

**Issue by jlamande** [aquasecurity/starboard#831]
**What steps did you take and what happened:**

Currently deploying the starboard operator on ARM instances is not working (e.g. Graviton AWS EC2)

On start :
```
standard_init_linux.go:228: exec user process caused: exec format error
```

This error is a known symptom of not supported arm architecture.

**What did you expect to happen:**

Operator starting.

**Anything else you would like to add:**

Blocked on operator deployment, I have not checked on tools used by Starboard if the same issue will happen (kube bench, trivy, polaris, ...)

**Environment:**

- Starboard version (use `starboard version`): 0.13.0
- Kubernetes version (use `kubectl version`): 1.22
- OS (macOS 10.15, Windows 10, Ubuntu 19.10 etc): EKS 1.21, Graviton t4g, Bottlerocket OS

**Comment by jlamande** [comment]
Hi @danielpacak 

I created a PR for adding ARM support.
_Hard to validate as it depends on github actions and releases._

**Comment by danielpacak** [comment]
> Hi @danielpacak
> 
> I created a PR for adding ARM support. _Hard to validate as it depends on github actions and releases._

To test your changes before you can always install goreleaser on your computer as explained on https://goreleaser.com/install/

And then just test it out with the following command:

```
goreleaser release --snapshot --skip-publish --rm-dist
```

**Comment by danielpacak** [comment]
@jlamande I've renamed this issue to indicate that we want to build Starboard CLI for ARM architectures as well.

**Comment by chen-keinan** [comment]
Fixed in PR #955
